meshtastic/ota.py

```
@@ -97,44 +127,62 @@ def update(
+                file_hash.hexdigest(),
+            )
+
+            self._socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            self._socket.settimeout(OTA_SOCKET_TIMEOUT_SECONDS)
+            try:
+                self._socket.connect((self._hostname, self._port))
+                logger.debug("Connected to %s:%d", self._hostname, self._port)
+
+                # Send start command
+                self._socket.sendall(
+                    f"OTA {size} {file_hash.hexdigest()}\n".encode("utf-8")
```

Ruff (0.15.4)

[warning] Unnecessary UTF-8 `encoding` argument to `encode`

Remove unnecessary `encoding` argument

(UP012)